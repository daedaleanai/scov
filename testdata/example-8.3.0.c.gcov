version:8.3.0
file:example.c
function:28,59,1,main
lcount:28,1,0
lcount:34,1,0
lcount:36,1,0
lcount:37,1,0
lcount:43,1,0
lcount:44,1,0
lcount:49,1,0
lcount:51,0,1
lcount:55,1,0
lcount:58,1,0
