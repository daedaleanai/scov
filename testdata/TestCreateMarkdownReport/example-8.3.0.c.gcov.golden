# SCov

## Metadata

Date: Mon Jan  2 15:04:05 UTC 2006


## Coverage Summary

|        | Hits   | Total  | Coverage |
| :----- | :----: | :----: | :------: |
| Lines: | 9 | 10 | 90.0% |
| Functions: | 1 | 1 | 100.0% |


## By File

| Filename | Line Coverage | Function Coverage |
| :------- | :-----------: | :---------------: |
| example.c | 9/10 (90.0%) | 1/1 (100.0%) |


## By Function

| Function | Hits |
| :------- | :--: |
| main | 1 |


***
Generated by [SCov](https://gitlab.com/stone.code/scov).

